SConstruct

```diff
@@ -2151,14 +2148,14 @@ if not GetOption('help'):
         subdirs += ['examples']
 
     if 'tests' in COMMAND_LINE_TARGETS:
-        if env['NOSETESTS'] == '':
-            nosetests = which('nosetests')
-            if nosetests is None:
-                env['NOSETESTS'] = None
+        if env['PYTEST'] == '':
+            pytest = which('pytest')
+            if pytest is None:
```

Comment:
I think it would be worth adding a check for `py.test` at this point.  Apparently, the `pytest` executable name is newish -- version 3.0 -- so if it doesn't exist, it would be nice to check for `py.test` automatically rather than fail.

I didn't even know that `pytest` was valid, since they are keeping the `py.test` executable around still, just deprecated:
> [...] so from pytest 3.0 we will supported and recommend use of pytest as the main command instead of py.test. It's possible that in future we will deprecate py.test and potentially even remove it.